packages/react/src/components/Dialog/index.tsx

```diff
@@ -42,6 +43,7 @@ export default class Dialog extends React.Component<DialogProps, DialogState> {
 
   static propTypes = {
     className: PropTypes.string,
+    headerClassName: PropTypes.string,
```

Comment:
Why can't we use the className from the dialog?

Reply:
looks like css module's `:global` will solve my specific problem so I am declining this PR